samples/multi-kotlin-project-config-injection/build.gradle.kts

```diff
@@ -16,9 +16,17 @@ allprojects {
     }
 }
 
-// Configure all KotlinCompile tasks on each sub-project
+// Apply and configure the Kotlin Gradle plugin on each sub-project
 subprojects {
 
+    apply {
```

Comment:
We should make sure `cli/build.gradle.kts` can still be edited with the correct classpath when this sample is imported in IntelliJ.